config.yml

```diff
@@ -543,10 +543,35 @@ nodes:
     fields:
       - name: key
         type: node
+        comment: |
+          The key of the association. This can be any node.
+
+              { a: b }
+                ^
+
+              { foo => bar }
+                ^^^
+
+              { def a; end => 1 }
+                ^^^^^^^^^^
       - name: value
         type: node?
+        comment: |
+          The value of the association, if present. This can be any node. Will
+          not be present if, e.g., this node is an element of a `HashPatternNode`.
```

Comment:
```suggestion
          The value of the association, if present. This can be any node that
          represents a non-void expression. It can be optionally omitted if this
          node is an element in a `HashPatternNode`.
```